be/src/util/faststring.h

```diff
@@ -82,7 +82,7 @@ class faststring {
     OwnedSlice build() {
         uint8_t* ret = data_;
         if (ret == initial_data_) {
-            ret = new uint8_t[len_];
+            ret = (uint8_t*)malloc(len_);
             memcpy(ret, data_, len_);
```

Comment:
ownedslice deconstructor using delete[]  to release data_, if we use malloc here, we need to modify it.